src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala

```diff
@@ -10,22 +10,34 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.Row
 import org.slf4j.LoggerFactory
 
+import scala.util.Try
 import scala.util.control.Exception
 
 @Singleton
 class PopulationParser @Inject()(districtsService: DistrictsService,
                                  districtsStorage: DistrictsStorage) {
   private lazy val logger = LoggerFactory.getLogger(getClass)
+  private lazy val POPULATION_YEAR = 2010
```

Comment:
Does it mean that you have population data only for 2010?

Reply:
Yes, because our dataset is only for 2000 and 2010

Reply:
Hmm, it's a kind of restriction... General speaking you don't generate stats for [2000;2009] years?

Reply:
Yes, we are interested in only in a latest information.

Reply:
Ok, I see. Thanks